src/sql/src/plan/statement/ddl.rs

```diff
@@ -1348,6 +1365,23 @@ pub fn plan_create_sink(
     })
 }
 
+/// Returns only those `CatalogItem`s that don't have any other user
+/// dependencies. Those are the root dependencies.
+fn get_root_user_dependencies<'a>(
```

Comment:
This function name is a little odd because it could be read as "get [root user] dependencies", i.e. get the root user's dependencies, as opposed to root dependencies in userland. I was very confused why sinks needed "root user" dependencies at first.

Maybe just `get_root_dependencies`?

Reply:
Will do!